Cpp 17 issue with FairMQ 1.4.8  und Apple clang version 12.0.0 (clang-1200.0.26.2)

**Describe the bug**
FairMQ does not compile with Apple clang version 12.0.0 (clang-1200.0.26.2)
 
**To Reproduce**
Steps to reproduce the behavior:

MacOS 10.15.6
Apple clang version 12.0.0 (clang-1200.0.26.2)
FairMQ 1.4.8 or later 

**Output**
```

/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/memory:1720:13: error: static_assert failed due to requirement
      'is_cpp17_move_insertable<std::1::allocator<FairMQChannel>, false>::value' "The specified type does not meet the requirements of Cpp17MoveInsertable"
            static_assert(is_cpp17_move_insertable<allocator_type>::value,
            ^             ~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/vector:952:21: note: in instantiation of function template specialization
      'std::1::allocator_traits<std::__1::allocator<FairMQChannel> >::construct_backward_with_exception_guarantees<FairMQChannel *>' requested here
    alloc_traits::construct_backward_with_exception_guarantees(
                    ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/vector:1668:5: note: in instantiation of member function 'std::1::vector<FairMQChannel,
      std::__1::allocator<FairMQChannel> >::swap_out_circular_buffer' requested here
    swap_out_circular_buffer(v);
    ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/vector:1686:9: note: in instantiation of function template specialization
      'std::1::vector<FairMQChannel, std::__1::allocator<FairMQChannel> >::emplace_back_slow_path<const std::1::basic_string<char> &, int &,
      std::1::map<std::1::basic_string<char>, boost::any, std::1::less<std::1::basic_string<char> >, std::1::allocator<std::1::pair<const
      std::__1::basic_string<char>, boost::any> > > >' requested here
        emplace_back_slow_path(_VSTD::forward<_Args>(args)...);
        ^
/Users/turany/development/FairSoft/legacy/basics/FairMQ/fairmq/FairMQDevice.cxx:285:35: note: in instantiation of function template specialization
      'std::1::vector<FairMQChannel, std::1::allocator<FairMQChannel> >::emplace_back<const std::__1::basic_string<char> &, int &,
      std::1::map<std::1::basic_string<char>, boost::any, std::1::less<std::1::basic_string<char> >, std::1::allocator<std::1::pair<const
      std::__1::basic_string<char>, boost::any> > > >' requested here
            fChannels[info.first].emplace_back(info.first, i, fConfig->GetPropertiesStartingWith(tools::ToString("chans.", info.first, ".", i, ".")));
                                  ^
1 error generated.
make[2]: *** [fairmq/CMakeFiles/FairMQ.dir/FairMQDevice.cxx.o] Error 1
```

--- a/fairmq/FairMQChannel.h
+++ b/fairmq/FairMQChannel.h
@@ -66,13 +66,13 @@ class FairMQChannel
     FairMQChannel(const FairMQChannel&, const std::string& name);
 
     /// Move constructor
-    FairMQChannel(FairMQChannel&&) = delete;
+    // FairMQChannel(FairMQChannel&&) = delete;
 
     /// Assignment operator
     FairMQChannel& operator=(const FairMQChannel&);
 
     /// Move assignment operator
-    FairMQChannel& operator=(FairMQChannel&&) = delete;
+    // FairMQChannel& operator=(FairMQChannel&&) = delete;
 
     /// Destructor
     virtual ~FairMQChannel()

